docs/start/framework/testing.md

```diff
@@ -13,7 +13,7 @@ Consider a login form component that relies on `useActionData`
 import { useActionData } from "react-router";
 
 export function LoginForm() {
-  const errors = useActionData();
+  const errors = useActionData()?.errors;
```

Comment:
Let's go with a destructure to not repeat the `errors` and make it easier for users to add extra properties they want to use

```suggestion
  const { errors } = useActionData();
```